### ISSUE TITLE
PageSpeed metrics (esp. "Time to Interactive") are massively larger than real-world experience

### ISSUE THREAD

**Issue by diamondavocado** [GoogleChrome/lighthouse#9995]
#### Provide the steps to reproduce
1. Run LH on a variety of non-static websites. Examples:
a. [https://uk.burberry.com/](https://developers.google.com/speed/pagespeed/insights/?url=https%3A%2F%2Fuk.burberry.com%2F)
b. [https://www.johnlewis.com/](https://developers.google.com/speed/pagespeed/insights/?url=https%3A%2F%2Fwww.johnlewis.com%2F)
c. [https://www.diy.com/](https://developers.google.com/speed/pagespeed/insights/?url=https%3A%2F%2Fwww.diy.com%2F)
d. [https://www.harrods.com](https://developers.google.com/speed/pagespeed/insights/?url=https%3A%2F%2Fwww.harrods.com%2F)

#### What is the current behavior?
Performance timings/figures are hugely overblown compared to real-world in-browser experience. All of the above websites fully load in my browser (to interactive) in under 2.5 seconds and their first content paint is under 1 second. 

However PageSpeed Insights reports a "Time to Interactive" of **between 15 and 32 seconds** for all of the sites I test which is basically nonsense. This is totally at odds with the real-world experience of these websites.

**Edit:** I note that the Github readme specifically states that "Time to Interactive" is in beta, but it also has the highest weighting of all the metrics and is therefore skewing the overall performance score massively. If it's in beta, it shouldn't have such a heavy weighting. 

Non-technical people (i.e. managers) who run PageSpeed Insights tests don't care if metrics are in beta; they just look at the overall score and say "our website is slow". Lighthouse, and by extension, PageSpeed Insights, is essentially useless at the moment. 

#### What is the expected behavior?
Figures should be more realistic and represent real-world use.

#### Environment Information
PageSpeed Insights

**Comment by paulirish** [comment]
> All of the above websites fully load in my browser......

PageSpeed and Lighthouse are designed to report not on the "typical" user's experience but on the "typical bad" experience. Generally we consider the 85th percentile real-world experience to be the benchmark for what we're emulating.

A nexus 5x or moto g4 with [this network situation](https://github.com/GoogleChrome/lighthouse/blob/master/docs/throttling.md#the-mobile-network-throttling-preset) is our baseline.


I took a closer look at harrods and I'm generally happy with what i'm seeing.

1. PSI reports a a TTI of 17s
1. WebPageTest reports a "first CPU idle" (which is generally the same or a little earlier than TTI) [of 19.3s](https://webpagetest.org/result/191120_F8_76cbb22ebd314ee5e62ac805b19e2944/) when run on a real Moto G4.

The page doesn't do much work on the main thread! 🎉  (except for some surprising amount of jQuery navmenu building). 

But, the page **loads 5 megabytes of data**.  This is exactly what Lighthouse is flagging. On a slower network connection, this takes a very long time to download.   Most of the weight is images, but a 1.1MB video is also in there. The images and video could all be significantly smaller without any loss in visual fidelity.

On wifi, this page doesn't load slowly, which is why you'll see PSI reports a TTI of 3.6s for desktop. That seems fair.

**Comment by diamondavocado** [comment]
@paulirish Thanks for your reply and the explanation. 

I'm not sure if you are personally involved the Google PageSpeed project so please forgive me if I'm addressing the wrong person. 

But if anyone from that team will see this discussion, can I please strongly suggest that you make it crystal clear on the PageSpeed page that the tests are being run with that ("typical bad") baseline? 

At the moment what's happening is that users are going to the PageSpeed website, testing their own websites and then panicking because their performance scores are terrible. That results in a flurry of angry emails to developers (like me) from non-technical people who are annoyed because "Google says our website only scores 5/100!"

There's nothing on the page to explain that the score is calculated based off a simulation of a poor connection, so I can't blame these managers for reacting this way. 

Furthermore, PageSpeed used to use a different set of scoring rules and these websites used to score much higher than they do now. That's also going to panic people because "our website used to be 80/100 and now it's gone down to 8!!!!!!!!"

**TL;DR** It's critical that PageSpeed needs to communicate the context of these tests better to the end-user.

**Comment by peixotorms** [comment]
> PageSpeed and Lighthouse are designed to report not on the "typical" user's experience but on the "typical bad" experience. Generally we consider the 85th percentile real-world experience to be the benchmark for what we're emulating.
> 
> A nexus 5x or moto g4 with [this network situation](https://github.com/GoogleChrome/lighthouse/blob/master/docs/throttling.md#the-mobile-network-throttling-preset) is our baseline.

```
The mobile network throttling preset
This is the standard recommendation for mobile throttling:

Latency: 150ms
Throughput: 1.6Mbps down / 750 Kbps up.
Packet loss: none.
These exact figures are used as Lighthouse's throttling default and represent roughly the bottom 25% of 4G connections and top 25% of 3G connections (In Lighthouse it is sometimes called "Slow 4G" used to be labeled as "Fast 3G"). This preset is identical to the WebPageTest's "Mobile 3G - Fast" and, due to a lower latency, slightly faster for some pages than the WebPageTest "4G" preset.
```

May I ask, why is a 2016, slow phone used for benchmarking?
In reality, how many people are even using that phone around the world?
Any public statistics to show that this is the average performance of a typically bad experience?

Also, a throughput of 1.6Mbps down / 750 Kbps up doesn't seem correct, by 2019/2020 standards.
I travel frequently through countries which are considered less developed, and I hardly ever see speeds lower than 10 Mbps throughout southeast asia and other regions.

In fact, in 2009 (ten years ago), the baseline for mobile speed was estimated at 7.2 Mbps around the world.
https://www.3gpp.org/news-events/partners-news/1231-gsa-confirms-7-2-mbps-as-new

And now, in 2019 it's going to be 21 Mbps.
https://mybroadband.co.za/news/wireless/10521-21-mbps-to-be-next-mobile-broadband-baseline.html

I don't think Google tests should be stuck on average speeds from 10+ years ago.
It also doesn't make sense to keep using a Moto G4 for tests (which nobody is using nowadays).
Even in low income countries, brands such as xiaomi, oppo, etc offer very good phones with latest snapdragon processors and plenty of ram.
In fact, the poorer the country, the more it seems that people want to have the latest, fastest gadget they can afford.

I would suggest that lighthouse tests start using at least 7.2 Mbps speeds, and at the very least, a Google Pixel, or Google Pixel 2 performance as a benchmark.

**Comment by connorjclark** [comment]
In the US, the [average replacement cycle](https://www.statista.com/statistics/619788/average-smartphone-life/) for phones is nearly 3 years.

According to a [recent analysis](https://almanac.httparchive.org/en/2019/mobile-web), the median mobile user is using a ~$100 smart phone with a connection of 0.4 - 1.6Mbps.

If you feel these constraints don't match what you believe to be your median user, I suggest you defer to the desktop report in PSI.

**Comment by peixotorms** [comment]
@connorjclark  thanks for the input.

The average replacement cycle for phones at nearly 3 years, makes sense and I would assume it's correct. However, we have to be realistic. 

If a person buys a premium phone, it makes sense that they want to hold on to it for 3 years. However, if a user bought a cheap phone, I doubt that the average user will keep it for 3 years.

It "should be" simple for the lighthouse team to cross information with the Google Play team, and get meaningful information about which android devices are still being used... after all, you (Google) already own that data.

That data from statista.com doesn't justify the fact that Google is using a Moto G4 or Nexus 5. 
I would still agree with using an old device, but it should be something reasonable.
A popular device, or one of the devices that offer good value for users to keep them that long, such as the Google Pixel 1 (which is also a 2016 phone), would be reasonable.

I would say, the likelihood of a user holding on to a google pixel 1 phone for 3 years, is much higher than owning a G4 in 2019/2020. It's common sense.

However, according to some benchmarks, a Google Pixel 1 is roughly 4 times faster than a Moto G4.
https://www.ubergizmo.com/products/lang/en_us/devices/moto-g4,google-pixel/

In addition, an average 1.6 Mbps speed doesn't look realistic to me, no matter what that recent analysis shows. There are others, probably more reliable which suggest otherwise, such as https://www.speedtest.net/reports/united-states/

With the advent of 5G next year, I believe at least some internal talks are due within the team.

**Comment by Roaders** [comment]
For my site (a large angular project) I am seeing times of around 60 seconds for all performance metrics other than first input delay when in reality it's more like 3 seconds. Max Potential First input delay is 3 seconds - this seems more accurate for all the metrics.

I am developing internal applications for a company. I know that everyone is on modern powerful machines and on a local network. Can I configure the cpu profile that is used to create these metrics?

Thanks

**Comment by paulirish** [comment]
@peixotorms we approximate a 85th percentile device and connectivity. We are comfortable with our throttling presets right now.  That topic seems like a separate issue however.

@Roaders with the CLI and/or configuration you can customize all the throttling settings.

As for the original topic, I showed that Lighthouse metric results are accurate.

**Comment by radogado** [comment]
LH reports 18" time to interactive, while it's <1", also according to the trace from the same report. My site doesn't need JS at all to be interactive. Very misleading for metrics-obsessed people.

**Comment by patrickhulce** [comment]
@radogado that's almost definitely #9971

**Comment by intellix** [comment]
don't want to go off topic but have yet to see an explanation about the times. It seems misleading that it's shown in seconds when the test is clearly not even running for that period of time.

Upon clicking "Generate Report" the site is ready almost instantly (Angular Universal fully delivered response), then about 3 seconds later I get the report that the FCP and LCP are 10 - 15seconds. How can it say that it took 15 seconds if the test was over in 3?

Are the timings inflated due to network throttling?

**Comment by patrickhulce** [comment]
@intellix see https://github.com/GoogleChrome/lighthouse#why-is-the-performance-score-so-low-it-looks-fine-to-me
